tests/test_transforms.py

```diff
@@ -478,9 +478,9 @@ def test_longest_max_size_keypoints():
     result = aug(image=img, keypoints=keypoints)
     assert result["keypoints"] == [(18, 10, 0, 0)]
 
-    aug = A.LongestMaxSize(max_size=5, p=1)
```

Comment:
Please add independent tests for the case when `max_size` is list

Reply:
Updated tests